lib/main_common.pm

```diff
@@ -2490,7 +2490,7 @@ sub load_publiccloud_tests {
     if (get_var('PUBLIC_CLOUD_PREPARE_TOOLS')) {
         loadtest "publiccloud/prepare_tools";
     }
-    elsif (get_var('PUBLIC_CLOUD_IPA_TESTS') or get_var('PUBLIC_CLOUD_CHECK_BOOT_TIME')) {
+    elsif (get_var('PUBLIC_CLOUD_IPA_TESTS')) {
```

Comment:
instead going this direction ( separate variable for each test ) maybe we should do something similar as we have for wicked ? - reuse of extratests load infrastructure ?

Reply:
we will move it to YAML in another PR